loader/icd_dispatch_generated.c

```diff
@@ -19,6 +19,8 @@
 #include "icd_dispatch.h"
 #include "icd.h"
 
+#include <string.h>
+
 #ifdef __cplusplus
```

Comment:
This file is generated, so it shouldn't be modified. Also I don't understand why this include would suddenly be required.
If the modification was required, https://github.com/KhronosGroup/OpenCL-ICD-Loader/blob/main/scripts/icd_dispatch_generated.c.mako would need to be updated before regenerating the dispatch.
```suggestion
#ifdef __cplusplus
```

Reply:
Fixed.  thanks